ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml

```diff
@@ -62,31 +62,51 @@ tests:
       TEST_TYPE: upgrade
     workflow: baremetalds-e2e-upgrade-ovn-ipv6
 - as: e2e-metal-ipi-serial-ipv4
-  skip_if_only_changed: (^(agent|docs|.github)/)|^config_example.sh$|\.md$|^(DCO|LICENSE|OWNERS)$
+  skip_if_only_changed: (^(agent|docs|.github)/)|^config_example.sh$|\.md$|^(DCO|LICENSE)$|((^|/)OWNERS$)
   steps:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e-serial-ipv4
 - as: e2e-metal-ipi-serial-ovn-ipv6
-  skip_if_only_changed: (^(agent|docs|.github)/)|^config_example.sh$|\.md$|^(DCO|LICENSE|OWNERS)$
+  skip_if_only_changed: (^(agent|docs|.github)/)|^config_example.sh$|\.md$|^(DCO|LICENSE)$|((^|/)OWNERS$)
   steps:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e-serial-ovn-ipv6
 - as: e2e-metal-ipi-virtualmedia
   optional: true
-  skip_if_only_changed: (^(agent|docs|.github)/)|^config_example.sh$|\.md$|^(DCO|LICENSE|OWNERS)$
+  skip_if_only_changed: (^(agent|docs|.github)/)|^config_example.sh$|\.md$|^(DCO|LICENSE)$|((^|/)OWNERS$)
   steps:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e-virtualmedia
 - as: e2e-metal-ipi-proxy-ipv4
-  skip_if_only_changed: (^(agent|docs|.github)/)|^config_example.sh$|\.md$|^(DCO|LICENSE|OWNERS)$
+  skip_if_only_changed: (^(agent|docs|.github)/)|^config_example.sh$|\.md$|^(DCO|LICENSE)$|((^|/)OWNERS$)
   steps:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e-proxy-ipv4
 - as: e2e-metal-ipi-proxy-ipv6
-  skip_if_only_changed: (^(agent|docs|.github)/)|^config_example.sh$|\.md$|^(DCO|LICENSE|OWNERS)$
+  skip_if_only_changed: (^(agent|docs|.github)/)|^config_example.sh$|\.md$|^(DCO|LICENSE)$|((^|/)OWNERS$)
   steps:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e-proxy-ipv6
+- as: e2e-agent-compact
```

Comment:
This sounds as a good occasion to use the recently merged [agent compact workflow](https://github.com/openshift/release/blob/891e3bfe9294494fca96ead8be94c3fc85df54f6/ci-operator/step-registry/agent/e2e/compact/ipv4/agent-e2e-compact-ipv4-workflow.yaml#L2)

Reply:
Gave it a go but it doesn't have KNI_DEPLOY_FROM_GIT set :frowning_face:. We may need separate workflows for testing the installer and everything else until we are merged in the installer master branch.
In the meantime I copied the contents here - the predefined chains definitely make it simpler.

Reply:
Ack. If the `DEVSCRIPTS_CONFIG` it's the only part to be overriden (and unfortunately cannot be done partially), and the steps sequence remains exactly the same, then it should work also invoking directly a ` workflow: agent-e2e-compact-ipv4` (just to save a few lines of code)

Reply:
Done, thanks for the tip!